### PR TITLE
New HandBrake release 1.5.0

### PR DESCRIPTION
**Pull request by jstebbins**

### CODE REVIEW

**Comment by jstebbins** [comment]
bot, build

**Comment by jstebbins** [comment]
bot, build

File: fr.handbrake.ghb.json [hunk 1]
```diff
@@ -73,24 +74,24 @@
             ],
             "sources": [
                 {
-                    "url": "https://github.com/HandBrake/HandBrake/releases/download/1.4.2/HandBrake-1.4.2-source.tar.bz2",
-                    "sha256": "8b8e81b7dc2e3180f4e94e8c7f5337d2953f69f0d983ccce48096e29ed6dfb61",
+                    "url": "https://github.com/HandBrake/HandBrake/releases/download/1.5.0/HandBrake-1.5.0-source.tar.bz2",
```
Comment (Eonfge) [inline fr.handbrake.ghb.json:77]: You could try the git repo directly;

```
{
                    "type": "git",
                    "url": "https://github.com/HandBrake/HandBrake.git",
                    "tag": "1.5.0",
                    "commit": "a11651939c67347868d0cd31be771c027b45985c"
}
```

Comment (Eonfge) [reply fr.handbrake.ghb.json:77]: Fair, if you can get it fixed upstream, that would be great for everybody

Comment (Eonfge) [reply fr.handbrake.ghb.json:77]: Ow great. Best wishes and thanks for your work on Handbrake and Flathub

**Pull request closed without merging**